To: vim_dev@googlegroups.com
Subject: Patch 7.3.157
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.157
Problem:    Superfluous assignment.
Solution:   Remove assignment.
Files:	    src/misc1.c


*** ../vim-7.3.156/src/misc1.c	2011-03-22 13:07:19.000000000 +0100
--- src/misc1.c	2011-04-11 14:11:17.000000000 +0200
***************
*** 6773,6780 ****
  		{
  		    curwin->w_cursor.lnum = our_paren_pos.lnum;
  		    curwin->w_cursor.col = col;
! 		    if ((trypos = find_match_paren(ind_maxparen,
! 						     ind_maxcomment)) != NULL)
  			amount += ind_unclosed2;
  		    else
  			amount += ind_unclosed;
--- 6775,6781 ----
  		{
  		    curwin->w_cursor.lnum = our_paren_pos.lnum;
  		    curwin->w_cursor.col = col;
! 		    if (find_match_paren(ind_maxparen, ind_maxcomment) != NULL)
  			amount += ind_unclosed2;
  		    else
  			amount += ind_unclosed;
*** ../vim-7.3.156/src/version.c	2011-04-11 14:24:33.000000000 +0200
--- src/version.c	2011-04-11 14:25:36.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     157,
  /**/

-- 
A radioactive cat has eighteen half-lives.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
